draft-wussler-openpgp-pqc.md

```diff
@@ -976,24 +976,28 @@ scheme is as follows:
 
 ### Public-Key Encrypted Session Key Packets (Tag 1) {#ecc-kyber-pkesk}
 
-The composite Kyber algorithms MUST be used only with v6 PKESK, as defined in
-[I-D.ietf-openpgp-crypto-refresh] Section 5.1.2.
+The algorithm-specific fields consists of:
 
-The algorithm-specific v6 PKESK parameters consists of:
+ - The one-octet algorithm identifier, if it was passed (in the case of a v3 PKESK packet).
 
  - A fixed-length octet string representing an ECC ephemeral public key in the
    format associated with the curve as specified in {{ecc-kem}}.
 
  - A fixed-length octet string of the Kyber ciphertext in native format, whose
    length depends on the algorithm ID as specified in {{tab-kyber-artifacts}}.
 
- - A variable-length field containing the symmetric key:
+ - A variable-length field containing the wrapped session key:
 
    - A one-octet size of the following field;
 
-   - Octet string of the wrapped symmetric key as described in
+   - The wrapped session key represented as an octet string, i.e.,
+     the output of the encryption procedure described in
      {{ecc-kyber-encryption}}.
 
+Note that unlike ECDH, no checksum or padding are appended to the session key before key wrapping.
```

Comment:
I would not mention the differences here, this is an algorithm specific part. Let's just state how it's done here. Also AFAIK this is not done already in the new X25519 and X448 algos

Reply:
Yeah, but here it's not X25519 vs ECDH, here is a brand new algo. I don't know if it makes sense to add this non-normative sentence when the behavior is specified 3 lines above and the test vectors will make it clear that there is no key.

Reply:
I think we can mark this older conversation now as resolved (required for merging).